test/mailers/previews/developer_mailer_preview.rb

```diff
@@ -0,0 +1,6 @@
+class DeveloperMailerPreview < ActionMailer::Preview
+  def flagged
```

Comment:
```suggestion
  def inivisiblize
```

Reply:
Agreed. I dont know why I went with `flagged` and I only caught it after I opened the PR.